### PR TITLE
Migrate topics out of OTHER CAPABILITIES

### PR DESCRIPTION
**Pull request by rhetoric101**
This is work for NR-262987

### CODE REVIEW

File: src/nav/infrastructure.yml [hunk 1]
```diff
@@ -142,6 +142,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration
           - title: Aerospike integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration
+          - title: Monitor Apache Airflow with OpenTelemetry
```
Comment (paperclypse) [inline src/nav/infrastructure.yml:145]: Seems odd the way this is alphabetical order. Can we lose the "Monitor" here of shift it down the Ms?

File: src/nav/infrastructure.yml [hunk 2]
```diff
@@ -176,6 +188,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/hashicorp-consul-monitoring-integration
           - title: IBM MQ monitoring integration
             path: /install/ibm-mq
+          - title: Monitor Jenkins with OpenTelemetry
```
Comment (paperclypse) [inline src/nav/infrastructure.yml:191]: Another one that seems oddly out of alphabetical

**Review by paperclypse — APPROVED** [review]
Looks good. Left a couple minor comments. Also, we'll need to update the homepage to match.

File: src/nav/infrastructure.yml (hunk 1 above)
Comment (rhetoric101) [reply src/nav/infrastructure.yml]: @paperclypse I removed "Monitor."

File: src/nav/infrastructure.yml (hunk 2 above)
Comment (rhetoric101) [reply src/nav/infrastructure.yml]: @paperclypse I removed "Monitor."

**Review by paperclypse — APPROVED** [review]
Looks good

**Review by paperclypse — APPROVED** [review]
Looks fine, as far as I can tell. I think you can ignore that failed vale linter check.

**Pull request merged**